Gruntfile.js

```diff
@@ -13,6 +13,14 @@ module.exports = function(grunt) {
 					'composer.json',
 					'CHANGELOG.md',
 					'LICENSE.md',
+
+					/*
+					 * Exclude files not necessary in the distribution.
+					 *
+					 * @link https://github.com/liquidweb/airstory-wp/issues/69
+					 */
+					'!includes/lib/wp-async-task/*.dist',
```

Comment:
That could also be a good approach, though I'll admit that holding SVN configuration in a Git repo feels funny. How do you feel about them being excluded in the build step *and* blocked at the `.svnignore` level only within the WordPress.org repo?